ql/src/java/org/apache/hadoop/hive/ql/exec/tez/YarnQueueHelper.java

```diff
@@ -51,41 +51,38 @@ public class YarnQueueHelper {
   private int lastKnownGoodUrl;
   private boolean sslForYarn;
   private boolean isHA;
-  private static String webapp_conf_key = YarnConfiguration.RM_WEBAPP_ADDRESS;
-  private static String webapp_ssl_conf_key = YarnConfiguration.RM_WEBAPP_HTTPS_ADDRESS;
-  private static String yarn_HA_enabled = YarnConfiguration.RM_HA_ENABLED;
-  private static String yarn_HA_rmids = YarnConfiguration.RM_HA_IDS;
 
   public YarnQueueHelper(HiveConf conf) {
     ArrayList<String> nodeList = new ArrayList<>();
     sslForYarn = YarnConfiguration.useHttps(conf);
-    isHA = conf.getBoolean(yarn_HA_enabled, false);
-    LOG.info(String.format("Yarn is using SSL: %s", sslForYarn));
-    LOG.info(String.format("Yarn HA is enabled: %s", isHA));
+    isHA = conf.getBoolean(YarnConfiguration.RM_HA_ENABLED, false);
+    LOG.info("Yarn is using SSL: " + sslForYarn);
```

Comment:
I believe we can merge these into a single log line:

```
 LOG.info("Yarn is using SSL: {}, HA enabled: {}", sslForYarn, isHA);
```

Reply:
OK, Done